packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -40,7 +40,10 @@ class OpPerfTelemetry {
 
         this.deltaManager.on("pong", (latency) => this.recordPingTime(latency));
         this.deltaManager.on("submitOp", (message) => this.beforeOpSubmit(message));
+
+        // Back-compat: <= 0.28: Replace to "op"" and remove "beforeOpProcessing" in the future.
         this.deltaManager.on("beforeOpProcessing", (message) => this.beforeProcessingOp(message));
```

Comment:
don't you need to register both here, and only this.beforeProcessingOp(message) if the second param !== undefined

Reply:
I can, but that would either double amount of work or I'd need to detect the case where both of them are sent.
I'd prefer to get version out as is, and then in next version change this code and delete old event - it's just simpler, and change is needed either way to clean it up.
Please let me know if I got it wrong.